Subject: [PATCH] build: Update Loom
---
Index: gradle/libs.versions.toml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/gradle/libs.versions.toml b/gradle/libs.versions.toml
--- a/gradle/libs.versions.toml	(revision ece17fdf9ee4a8bf3cadcec38e56b9cfd1f9013a)
+++ b/gradle/libs.versions.toml	(date 1767949911205)
@@ -25,7 +25,7 @@
 mixin = { module = "org.spongepowered:mixin", version = "0.8.7" }
 
 [plugins]
-fabricLoom = { id = "fabric-loom", version = "1.13-SNAPSHOT" }
+fabricLoom = { id = "net.fabricmc.fabric-loom", version = "1.14-SNAPSHOT" }
 modDevGradle = { id = "net.neoforged.moddev", version = "2.0.107" }
 forgeGradle = { id = "net.minecraftforge.gradle", version = "[6.0.25,6.2)" }
 mixin = { id = "org.spongepowered.mixin", version = "0.7-SNAPSHOT" }
